src/local/butler/appengine.py

```diff
@@ -120,7 +120,11 @@ def symlink_dirs():
   common.symlink(
       src=os.path.join('src', 'python'),
       target=os.path.join(SRC_DIR_PY, 'python'))
-  common.symlink(
+  # While importing third party modules, we may call pkg_resources.
+  # pkg_resources normalizes paths by calling os.path.realpath on them, which is
```

Comment:
you are right, realpath is now not allowed anymore :( https://github.com/google/clusterfuzz/issues/258#issuecomment-472287307